### PR TITLE
Merge `standalone-binaries` to the main CLI

### PR DESCRIPTION
**Pull request by Hennzau**
# Merge `standalon-binaries` inside `zfctl` CLI

This PR introduces improvements to the `zfctl` CLI  for better usability. 

## Daemon

The `zenoh-flow-standalone-daemon` crate has been removed and replaced by the CLI command: 

```bash
zfctl daemon start <NAME | CONFIGURATION>
```

## Runtime

The `zenoh-flow-standalone-runtime` crate has been removed and replaced by the CLI command:

```bash
zfctl run-local <dataflow.yml> <optional configuration>
```

### CODE REVIEW

File: rust-toolchain.toml [hunk 1]
```diff
@@ -1,2 +1,2 @@
 [toolchain]
-channel = "1.72.1"
+channel = "1.75"
```
Comment (J-Loudet) [inline rust-toolchain.toml:2]: A small comment on this: the version of the Rust toolchain is linked to the version of Zenoh. The reason being that Zenoh-Flow can be compiled to a Zenoh plugin and, in order for Zenoh to load the plugin, it needs to have been compiled with the same version of Rust.

Hence, if you want to tackle this part, I'd suggest to make a separate PR that does both: bump Zenoh to version 1.0.x and update the toolchain to 1.75 😉

(I did not look closely at the rest of your PR as you marked it as a draft)

Comment (Hennzau) [reply rust-toolchain.toml:2]: Oh thx! Yeah bumping Zenoh was also in my mind, I should definitely separate it in two PR.

**Comment by J-Loudet** [comment]
@Hennzau In order to accept your contribution, you need to create an account with the same e-mail address associated to your GitHub account on the Eclipse Foundation website and then sign the ECA. 😉
This page gives you all the information: https://www.eclipse.org/legal/eca/

**Comment by J-Loudet** [comment]
Thanks for the contribution @Hennzau!

I am currently traveling so I will be able to look at it thoroughly starting next week. There is however one aspect that caught my attention, you say:

> - Before this PR: Executing `zfctl runtime list` did not return any output.
> - Now: The command communicates with all active daemons and presents the running runtimes in a clear, organized format.

That is an odd behavior that I am unable to reproduce. I tried on my machine and that of a colleague and we both obtained the expected result.
Given we both use macOS, I'm wondering if it could be an OS specific issue. Could you explain what you tried and if you can reproduce it consistently?

**Comment by Hennzau** [comment]
> Thanks for the contribution @Hennzau!
> 
> I am currently traveling so I will be able to look at it thoroughly starting next week. There is however one aspect that caught my attention, you say:
> 
> > * Before this PR: Executing `zfctl runtime list` did not return any output.
> > * Now: The command communicates with all active daemons and presents the running runtimes in a clear, organized format.
> 
> That is an odd behavior that I am unable to reproduce. I tried on my machine and that of a colleague and we both obtained the expected result. Given we both use macOS, I'm wondering if it could be an OS specific issue. Could you explain what you tried and if you can reproduce it consistently?

Oh wait... I just realized I got the command wrong! I was actually referring to the `zfctl instance list` command. I will update my message

File: zfctl/src/instance_command.rs [hunk 2]
```diff
@@ -195,6 +196,36 @@ Caused by:
...
+                                    e
+                                ),
+                            }
+                        }
+                        Err(err) => tracing::error!("{:?}", err),
+                    }
+                }
+
+                println!("{table}");
+            }
```
Comment (Hennzau) [inline zfctl/src/instance_command.rs:228]: This is the code I added

File: examples/examples/topic-sender/src/main.rs
Comment (J-Loudet) [inline examples/examples/topic-sender/src/main.rs:1]: We already have an example that does exactly this in the Zenoh repository ([here](https://github.com/eclipse-zenoh/zenoh/blob/0.11.0/examples/examples/z_put.rs)). Given that you are inclined to add this, maybe more specific instructions are required in the README?

**Comment by Hennzau** [comment]
Hi! Thank you very much, I understood everything! I’ll clarify all of this.

**Comment by Hennzau** [comment]
It should be good now! Thx for the previous review :)

**Review by J-Loudet — CHANGES_REQUESTED** [review]
Sorry for the delay.

After taking some time to play with this new version of `zfctl` — having less command-line tools provides indeed a better experience — I don't think it makes sense to have `zfctl runtime` + `zfctl daemon` and I would suggest to fuse them.

I understand this goes against my previous feedback of "just merging", so that one is on me. :wink:
I can take care of it if you want.

**Comment by J-Loudet** [comment]
I have merged it as is and I will do the changes later on. ;-)

**Pull request merged**